client/components/BedRoomCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import LocalHotelIcon from '@material-ui/icons/LocalHotel';
+import WeekendIcon from '@material-ui/icons/Weekend';
+
+const SleepingArrangementCard = styled.div`
+  height: 85px;
+  width: 120px;
+  border-style: solid;
+  border-width: 2px;
+  border-color: #d7e6e8;
+  border-radius: 15px;
+  margin-right: auto;
+`;
+
+const IconDiv = styled.div`
+  margin-right: 50%;
+`;
+const BedroomCard = ({ bedroomDescription }) => {
+  const arrangementType = bedroomDescription.bedroomDescription
+    .split(' ')
+    .pop();
+  let BedIcon;
+  if (arrangementType === 'bed') {
+    BedIcon = <LocalHotelIcon />;
+  } else {
+    BedIcon = <WeekendIcon />;
+  }
+  return (
+    <SleepingArrangementCard>
+      <div className="sleepingArrangement">
+        <IconDiv>{BedIcon}</IconDiv>
+        <div className="header">{bedroomDescription.bedroomHeader}</div>
```

Comment:
is there a reason for the classname being valled header vs header tag?

Reply:
no. Including header tag seems kinda redundant no? Since the tag is the actual structure?